hooks/pre-command

```diff
@@ -26,6 +26,12 @@ done < <(env | sort)
 
 
 if [ "${#paths[@]}" -gt 0 ]; then
-  echo "--- Downloading artifacts"
-  buildkite-agent artifact download "$(IFS=, ; echo "${paths[*]}")" "${BUILDKITE_PLUGIN_ARTIFACTS_WORKDIR}"
+  build=${BUILDKITE_PLUGIN_ARTIFACTS_BUILD:-}
+  build_message=$([[ -n "$build" ]] && echo " from build: $build" || echo "")
+
+  echo "--- Downloading artifacts$build_message"
```

Comment:
I'd tend towards an if/else with duplicated echo lines for the two header types, it's more verbose, but the code will read better.